### ISSUE TITLE
Kitsu not maintaining uploaded QT's colour space for 

### ISSUE THREAD

**Issue by PeteDraper** [cgwire/kitsu#1146]
Kitsu is not maintaining the same Colour space for previews. Uploading a sample in Rec2020 ST 2084 converts to Rec709 with a major colour shift.

Steps to reproduce:
Upload aforementioned Rec2020 ST2084 sample as Revision, resulting generated QT is converted to Rec709 with colour shift.

Expected output: same colour space, just converted to h.264 for preview purposes

**Comment by frankrousseau** [comment]
Hello @PeteDraper, do you know if it's something possible with FFMPEG? We didn't dig into colorspaces yet with it.

**Comment by PeteDraper** [comment]
yes I believe so. FFMPEG can do wondrous things. basically, you just don't want to change the colour space of the source

**Comment by frankrousseau** [comment]
Ok we will see what we can do about it. If you can provide a test video, that would help.

**Comment by PeteDraper** [comment]
You can shoot anything with your iPhone on cinematic mode which outputs a Rec2020 (HLG2100) and upload that. The resulting Revision file doesn't tone map nicely and converts to a rec709. Would expect it to convert to a h.264 / h.265 but maintain the same colour space as the source material.

**Comment by tetsuoanimation** [comment]
As discussed on discord, here's a deepdive on ffmpeg and colorspaces from the canva blog.

https://www.canva.dev/blog/engineering/a-journey-through-colour-space-with-ffmpeg/

**Comment by mathbou** [comment]
Some additional links from the ASWF Media Review working-group, they try to establish good encoding practices for VFX review:

- https://richardssam.github.io/ffmpeg-tests/
- https://wiki.aswf.io/pages/viewpage.action?pageId=16031068

**Comment by frankrousseau** [comment]
It should work now. Feel free to reopen the issue if it is not the case. 

At some point we will had settings to manage compression and codecs.
